Dockerfile

```diff
@@ -1,39 +1,17 @@
 FROM python:3.10-bullseye
 
-ARG CLOUD_SDK_VERSION=452.0.0
-ENV GCLOUD_HOME=/home/google-cloud-sdk
-RUN apt-get update && \
-    apt-get install -y openjdk-11-jdk jq && \
-    apt-get clean && rm -rf /var/lib/apt/lists/*
-
-ENV PATH="$PATH:$GCLOUD_HOME"
-# required by google batch scripts
-RUN DOWNLOAD_URL="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz" \
-    && TMP_DIR="$(mktemp -d)" \
-    && curl -fL "${DOWNLOAD_URL}" --output "${TMP_DIR}/google-cloud-sdk.tar.gz" \
-    && mkdir -p "${GCLOUD_HOME}" \
-    && tar xzf "${TMP_DIR}/google-cloud-sdk.tar.gz" -C "${GCLOUD_HOME}" --strip-components=1 \
-    && "${GCLOUD_HOME}/install.sh" \
-    --bash-completion=false \
-    --path-update=false \
-    --usage-reporting=false \
-    --quiet \
-    && rm -rf "${TMP_DIR}"
-
-ENV PATH="$PATH:$GCLOUD_HOME/bin"
-RUN java -version
-
-# Set environment variables for Java
-ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64
-ENV PATH=$PATH:$JAVA_HOME/bin
-
-RUN pip install poetry==1.7.1
+RUN apt-get update \
+  && apt-get clean \
+  && apt-get install -y openjdk-11-jdk jq \
+  && rm -rf /var/lib/apt/lists/*
 
 ENV POETRY_NO_INTERACTION=1 \
     POETRY_VIRTUALENVS_IN_PROJECT=1 \
     POETRY_VIRTUALENVS_CREATE=1 \
-    POETRY_CACHE_DIR=/tmp/poetry_cache
+    POETRY_CACHE_DIR=/tmp/poetry_cache \
+    JAVA_HOME=/usr/bin/
 
+RUN pip install poetry==1.7.1
 WORKDIR /app
 
 COPY pyproject.toml poetry.lock ./
```

Comment:
Not the intention of this PR, just a question. Right now we are containerising practically the whole repo. The image would be lighter if we stuck to `src/gentropy` right?

Reply:
if we change the order in the release action (firstly publish to pypi and then create a tag in github) I think we might even drop the /app entirely, as we could install gentropy from pypi